extensions/skins/conversation_v1/Conversation.js

```diff
@@ -646,3 +646,41 @@ oppia.directive('progressDots', [function() {
     }]
   };
 }]);
+oppia.filter('sliceDots', function() {
```

Comment:
I have merged the behavior into progressDots directive.
This also made the logic implementation to be easy.
Done!.